examples/tracing-jaeger/src/main.rs

```diff
@@ -29,7 +29,7 @@ fn init_tracer() -> Result<opentelemetry_sdk::trace::Tracer, TraceError> {
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
-    let _tracer = init_tracer().expect("Failed to initialize tracer.");
+    let _tracer_provider = init_tracer().expect("Failed to initialize tracer provider.");
```

Comment:
need to set global here, else the rest of the code won't work!

Reply:
Updated.